Assets/MixedRealityToolkit.Tests/PlayModeTests/GltfTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema;
+using Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization;
+using NUnit.Framework;
+using System.Collections;
+using System.IO;
+using System.Threading.Tasks;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class GltfTests
+    {
+        const string relativePath = "Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Avocado/glTF/AvocadoCustomAttr.gltf";
```

Comment:
To make the tests more robust to file and folder name changes, would it be possible to load this path a different way, perhaps by GUID like in https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7441?

Reply:
Sounds good